warmenu.lua

```diff
@@ -524,27 +530,43 @@ function WarMenu.Display()
 			drawSubTitle()
 
 			currentKey = nil
+			local current_time = GetGameTimer()
 
-			if IsDisabledControlJustReleased(0, keys.down) then
-				PlaySoundFrontend(-1, 'NAV_UP_DOWN', 'HUD_FRONTEND_DEFAULT_SOUNDSET', true)
+			if IsDisabledControlPressed(0, keys.down) then
+				if (current_time - lastScroll[keys.down]) > 125 then
+					PlaySoundFrontend(-1, 'NAV_UP_DOWN', 'HUD_FRONTEND_DEFAULT_SOUNDSET', true)
 
-				if currentMenu.currentOption < optionCount then
-					currentMenu.currentOption = currentMenu.currentOption + 1
-				else
-					currentMenu.currentOption = 1
+					if currentMenu.currentOption < optionCount then
+						currentMenu.currentOption = currentMenu.currentOption + 1
+					else
+						currentMenu.currentOption = 1
+					end
+
+					lastScroll[keys.down] = current_time
 				end
-			elseif IsDisabledControlJustReleased(0, keys.up) then
-				PlaySoundFrontend(-1, 'NAV_UP_DOWN', 'HUD_FRONTEND_DEFAULT_SOUNDSET', true)
+			elseif IsDisabledControlPressed(0, keys.up) then
+				if (current_time - lastScroll[keys.up]) > 125 then
```

Comment:
Let's save `125` value as `scrollInterval` variable.

Reply:
Not sure if it's needed to modify this value by users, so I suggest to keep it as local variable.